app/components/sidebar/sidebar.html

```diff
@@ -40,6 +40,9 @@
     <li class="sidebar-list">
       <a ui-sref="volumes" ui-sref-active="active">Volumes <span class="menu-icon fa fa-cubes"></span></a>
     </li>
+    <li class="sidebar-list" ng-if="applicationState.endpoint.apiVersion >= 1.30 && applicationState.endpoint.mode.provider === 'DOCKER_SWARM_MODE'">
```

Comment:
It probably requires the manager role as well via `&& applicationState.endpoint.mode.role === 'MANAGER'"`.